# goal definition
std_msgs/Header header
int8 target_battery_level   # the maximum level of battery to charge
---
# result definition
std_msgs/Header header
bool goal_reached           # flag to confirm that the goal is reached
--- 
# feedback definition
std_msgs/Header header
int8 current_battery        # this has to be updated with frequency 1 Hz